### DISCUSSION TITLE
What is the mode-line configuration in the screenshots?

### DISCUSSION THREAD

**Discussion by leira** [rnkn/fountain-mode#133] [Q&A] [answered]
I'm really impressed by the screenshots:

![fountain-mode](https://raw.githubusercontent.com/rnkn/fountain-mode/master/screenshots/02.png)

I suppose the theme used here [contrib/fountain-theme.el](https://github.com/rnkn/fountain-mode/blob/master/contrib/fountain-theme.el). What about the mode-line configurations? I used the theme, but the mode-line stays the same.

**Comment by rnkn** [comment, marked as answer]
Hi @leira the theme is just my personal one here: https://git.paulwrankin.com/dotfiles/tree/emacs/stillpoint-theme.el

Only relevant to the mode-line is the function `stillpoint-balance-mode-line` then `mode-line-format`.

**Reply by leira** [reply]
Thanks a lot!

**Comment by leira** [comment]
BTW, what is the font used in the screenshot? I couldn't find the font settings in https://git.paulwrankin.com/dotfiles/.

**Reply by rnkn** [reply]
It's either Menlo or San Francisco Mono. Just the default on macOS.
